app-office/drawio-desktop/drawio-desktop-22.1.2.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="draw.io"
+MY_NODE_N="node-modules"
+MY_NODE_D="node_modules"
+DRAWIO_V="22.1.4"
+
+inherit desktop xdg
+
+DESCRIPTION="draw.io diagramming and whiteboarding desktop app"
+HOMEPAGE="https://www.drawio.com/"
+
+SRC_URI="
+	https://github.com/jgraph/drawio-desktop/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/jgraph/drawio/archive/refs/tags/v${DRAWIO_V}.tar.gz
+	https://github.com/MocioF/distfiles/raw/main/${PN}-${MY_NODE_N}-${PV}-${DRAWIO_V}.tar.xz
```

Comment:
I can't check right now, but from a glance this appears to be *all* the node_modules, pregenerated.

Including electron. Electron is a compiled binary so that would mean this distfile is arch specific. It would need to conditionalize that specifically, as well as avoid keywording arches that don't have a pregenerated distfile.